op-deployer/pkg/deployer/integration_test/apply_test.go

```diff
@@ -790,7 +806,7 @@ func TestIntentConfiguration(t *testing.T) {
 func runEndToEndBootstrapAndApplyUpgradeTest(t *testing.T, afactsFS foundry.StatDirFs, implementationsConfig bootstrap.ImplementationsConfig) {
 	lgr := implementationsConfig.Logger
 
-	ctx, cancel := context.WithTimeout(context.Background(), 90*time.Second)
+	ctx, cancel := context.WithTimeout(context.Background(), 180*time.Second)
```

Comment:
I think we should revert this.
```suggestion
	ctx, cancel := context.WithTimeout(context.Background(), 90*time.Second)
```

Reply:
Addressed [here](https://github.com/ethereum-optimism/optimism/pull/18593/commits/b413e89fa0d8908b05acea644c179f292883b691).